detectors/node/opentelemetry-resource-detector-azure/test/detectors/AzureFunctionsDetector.test.ts

```diff
@@ -102,3 +102,26 @@ describe('AzureFunctionsDetector', () => {
     );
   });
 });
+
+it('should detect azure functions if websiteSku is defined as FlexConsumption', () => {
+  process.env.WEBSITE_SITE_NAME = 'test-service';
+  process.env.REGION_NAME = 'test-region';
+  process.env.WEBSITE_INSTANCE_ID = 'test-instance-id';
+  process.env.WEBSITE_SKU = 'FlexConsumption';
+  process.env.WEBSITE_MEMORY_LIMIT_MB = '1000';
+  process.env.WEBSITE_OWNER_NAME = 'test-owner-name';
+  process.env.WEBSITE_RESOURCE_GROUP = 'test-resource-group';
```

Comment:
can you make sure that the value of `FUNCTIONS_VERSION` is unset (just in case it gets added in another test by accident and your test starts with it already setup), so we know the results you're getting is because is doing the proper check on the `WEBSITE_SKU` and not because it has a function version?

Reply:
Added an assertion to ensure that these values are not defined before this test.